samples/graph-appcatalog-lifecycle/csharp/AppCatalogSample/Helper/AppCatalog.cs

```diff
@@ -0,0 +1,394 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using AdaptiveCards;
+
+namespace AppCatalogSample.Helper
+{
+  public class AppCatalog
+    {
+        public static  string gToken { get; set; }
+        public string GetToken(string token)
```

Comment:
This should be SetToken(). Could you please remove static method and have it as instance property (May be constructor parameter)?